packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_SequencerEntrypoint.sol

```diff
@@ -21,7 +21,6 @@ contract OVM_SequencerEntrypoint {
 
     using Lib_EIP155Tx for Lib_EIP155Tx.EIP155Tx;
 
-
```

Comment:
nit: this removal is inconsistent with most other solidity files, including `OVM_ECDSAContractAccount`

Reply:
Sorry, shouldn't be making unnecessary diffs